sttp/client/src/main/scala/endpoints4s/sttp/client/Endpoints.scala

```diff
@@ -37,7 +21,8 @@ import _root_.sttp.client3.{
   */
 class Endpoints[R[_]](
     val host: String,
-    val backend: SttpBackend[R, Any]
+    val backend: SttpBackend[R, Any],
+    val timeout: Option[FiniteDuration] = None
```

Comment:
You will probably have to re-introduce the old constructor signature as a secondary constructor here as well.

Reply:
It has been implemented